AWS/aws_redshift_get_ec2_daily_costs.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #53.        "secret_name": "\\"awsuser-doug-redshift\\""
</pre><p>this wont work, secret_name should be a parameter to this runbook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unskript/Awesome-CloudOps-Automation/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>